### DISCUSSION TITLE
关于observatory问题，不太懂

### DISCUSSION THREAD

**Discussion by GleenJi** [v2fly/v2ray-core#1695] [Q&A]
为什么error.log文件中会有这样的日志：
......
2022/03/26 00:43:33 [Warning] app/observatory/burst: error ping http://www.google.com/gen_204 with proxy1: Head "http://www.google.com/gen_204": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
......
2022/03/26 00:45:02 [Warning] app/observatory/burst: error ping http://www.google.com/gen_204 with proxy6: Head "http://www.google.com/gen_204": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
......

看样子像是ping http://www.google.com/gen_204失败了，想问问如果要换自定义的health_check URL，都有哪些URL啊？
还有api.v2fly.org，只要我设定自定义URL为什么会network is down？

**Comment by 4-FLOSS-Free-Libre-Open-Source-Software** [comment]
Your config of burst observatory?

Perhaps the HTTP status code matters?

https://github.com/v2fly/v2ray-core/pull/1991#issuecomment-1251928556
